openhtf/plugs/usb/fastboot_protocol.py

```diff
@@ -71,7 +70,7 @@ def send_command(self, command, arg=None):
     """
     if arg is not None:
       command = '%s:%s' % (command, arg)
-    self._write(io.StringIO(command), len(command))
+    self._write(six.StringIO(command), len(command))
```

Comment:
(I was curious about the difference--so using six.StringIO, the value under Python 3 must be unicode, but under Python 2, it doesn't have to be. Whereas when using io.StringIO, the value must always be unicode.)

Reply:
(It's also that the StringIO class was moved to the new module io in 3.x and the original modules were removed.)

Reply:
Right, but since `io` is available in 2.6+ I was just thinking in another use case we might have decided to use `io.StringIO` in hybrid 2/3 code rather than `six.StringIO`, since the behavior is slightly different.